### ISSUE TITLE
0.11D bip65-cltv test failure: JSONRPC error: CreateNewBlock(): TestBlockValidity failed

### ISSUE THREAD

**Issue by error10** [bitcoinxt/bitcoinxt#101]
```
Running 2nd level testscript bip65-cltv.py...
  File "/builddir/build/BUILD/bitcoinxt-0.11D/qa/rpc-tests/test_framework/test_framework.py", line 118, in main
    self.run_test()
  File "/builddir/build/BUILD/bitcoinxt-0.11D/qa/rpc-tests/bip65-cltv.py", line 31, in run_test
    self.nodes[1].generate(10)
  File "/builddir/build/BUILD/bitcoinxt-0.11D/qa/rpc-tests/test_framework/authproxy.py", line 126, in __call__
    raise JSONRPCException(response['error'])
Initializing test directory /tmp/testHaj57g
JSONRPC error: CreateNewBlock(): TestBlockValidity failed
Stopping nodes
Cleaning up
Failed
```

Possibly related to #93

**Comment by sandakersmann** [comment]
Ping @mikehearn and @gavinandresen

**Comment by dgenr8** [comment]
@error10 I can't reproduce when building from v0.11D tag.  What are you running exactly?

**Comment by error10** [comment]
@dgenr8 I'm building from this tarball https://github.com/bitcoinxt/bitcoinxt/archive/v0.11D.tar.gz

I'm building on Fedora 23 x86_64. The following packages were installed:

    dnf upgrade
    dnf install gcc gcc-c++ autoconf automake libtool libdb4-devel libdb4-cxx-devel qt-devel boost-devel openssl-devel libcurl-devel protobuf-devel

The build was done like this:

    ./autogen.sh
    ./configure
    make
    make check
    qa/pull-tester/rpc-tests.sh -extended

**Comment by dgenr8** [comment]
Run the test with --nocleanup and look at debug.log in the created /tmp/... directory.

**Comment by error10** [comment]
@dgenr8 I have node0, node1, node2 and node3. It looks like node0 and node2 started up fairly normally and seem to have nothing untoward. node3 never created a debug.log. And node1's debug.log showed nothing unusual except this:

```
2015-12-03 18:07:22 CreateNewBlock(): total size 1000
2015-12-03 18:07:22 ERROR: ContextualCheckBlockHeader : rejected nVersion=3 block
```

**Comment by dgenr8** [comment]
In the node1 log, what is the height= on the last UpdateTip message?

**Comment by dgenr8** [comment]
@error10 In node1 log, what is height= in the last UpdateTip message?

**Comment by error10** [comment]
@dgenr8 There are no UpdateTip messages in _any_ of the logs.

The version is:
2015-12-03 18:07:18 Bitcoin XT version v0.11.0D-gf2d5c61 (Sat, 28 Nov 2015 19:27:52 +0100)
2015-12-03 18:07:21 receive version message: /Bitcoin XT:0.11.0D/: version 70010, blocks=200, us=0.0.0.0:0, peerid=1

**Comment by error10** [comment]
[node0-regtest-debug.txt](https://github.com/bitcoinxt/bitcoinxt/files/51500/node0-regtest-debug.txt)
[node1-regtest-debug.txt](https://github.com/bitcoinxt/bitcoinxt/files/51499/node1-regtest-debug.txt)
[node2-regtest-debug.txt](https://github.com/bitcoinxt/bitcoinxt/files/51501/node2-regtest-debug.txt)

**Comment by error10** [comment]
Fixed in 0.11E.
